File: openapi-parser/src/main/java/io/openapiparser/ValidationMessage.java
package io.openapiparser;

public class ValidationMessage {
    // kind/source ?
    // warn/error ?
    // file ?
    private final String path;
    private final String text;

    public ValidationMessage (String path, String text) {
        this.path = path;
        this.text = text;
    }
}

File: openapi-parser/src/test/kotlin/io/openapiparser/support/ValidationMessage.kt
/*
 * Copyright 2021 https://github.com/openapi-processor/openapi-parser
 * PDX-License-Identifier: Apache-2.0
 */

package io.openapiparser.support

import io.openapiparser.validator.messages.ValidationMessage

@Deprecated("obsolete")
fun ValidationMessage.matches(path: String, text: String): Boolean {
    return this.path == path && this.text.contains(text)
}
